src/v3/src/transformer/layout/googleAuthenticator/transformGoogleAuthenticatorEnroll.test.ts

```diff
@@ -17,22 +17,28 @@ import {
   ButtonType,
   DescriptionElement,
   FieldElement,
+  FormBag,
   HeadingElement,
   QRCodeElement,
   StepperButtonElement,
   StepperLayout,
   TitleElement,
   WidgetProps,
 } from 'src/types';
+import { WidgetHooks } from 'src/util';
+import { TinyEmitter } from 'tiny-emitter';
 
 import { transformGoogleAuthenticatorEnroll } from '.';
 
 describe('Google Authenticator Enroll Transformer Tests', () => {
   const transaction = getStubTransactionWithNextStep();
-  const widgetProps: WidgetProps = {};
-  const formBag = getStubFormBag();
+  const widgetProps: WidgetProps = {
+    eventEmitter: null as unknown as TinyEmitter,
+    widgetHooks: null as unknown as WidgetHooks,
+  };
```

Comment:
Not sure what the best thing to do here is... It's throwing a TS error because these two properties are required on the type but this isn't a whole lot better. I also don't really think we _need_ to instantiate real versions of each since they aren't used in the tests. I guess I could write up some mock versions of both to be used in tests that would have spied methods to track calls if we wanted to do that.